lib/kubernetes-deploy/kubernetes_resource.rb

```diff
@@ -139,10 +140,11 @@ def debug_message
         if container_logs.blank? || container_logs.values.all?(&:blank?)
           helpful_info << "  - Logs: #{DEBUG_RESOURCE_NOT_FOUND_MESSAGE}"
         else
-          helpful_info << "  - Logs (last #{LOG_LINE_COUNT} lines shown):"
-          container_logs.each do |identifier, logs|
-            logs.split("\n").each do |line|
-              helpful_info << "      [#{identifier}]\t#{line}"
+          sorted_logs = container_logs.sort_by { |_, log_lines| log_lines.length }
+          sorted_logs.each do |identifier, log_lines|
+            helpful_info << "  - Logs from container '#{identifier}' (last #{LOG_LINE_COUNT} lines shown):"
+            log_lines.each do |line|
+              helpful_info << "      #{line}"
```

Comment:
Now looks like this:
![image](https://user-images.githubusercontent.com/4789493/27107502-53d04218-5066-11e7-9d5c-422928e6d307.png)
[...event noise addressed by other PR...]
![image](https://user-images.githubusercontent.com/4789493/27107480-3472125c-5066-11e7-8090-7133d815be91.png)